mesonbuild/backend/ninjabackend.py

```diff
@@ -1449,12 +1449,12 @@ def generate_compile_rule_for(self, langname, compiler, qstr, is_cross, outfile)
 '''
         else:
             command_template = ' command = %s %s $ARGS %s %s %s $in\n'
-        command = command_template % \
-            (' '.join([ninja_quote(i) for i in compiler.get_exelist()]),\
+        command = (command_template %
+            (' '.join([ninja_quote(i) for i in compiler.get_exelist()]),
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->